m2install.sh

```diff
@@ -1019,6 +1019,9 @@ index 1f454518e43..b3dcc784917 100644
       */
 -    const MAX_NUM_COOKIES = 50;
```

Comment:
This patch is corrupted, please change 
`@@ -29,7 +29,7 @@ class PhpCookieManager implements CookieManagerInterface`
to
`@@ -29,7 +29,10 @@ class PhpCookieManager implements CookieManagerInterface`

Reply:
Yeah, I missed that. Thank you!